.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -244,10 +242,15 @@ jobs:
       - name: Call all test APIs
         continue-on-error: true
         run: |
-          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/outgoing-http-call/
-          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/aws-sdk-call/
-          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/remote-service?ip=${{ env.REMOTE_SERVICE_POD_IP }}/
-          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/client-call/
+          curl -S -o /dev/null http://${{ env.APP_ENDPOINT }}/outgoing-http-call/
+          curl -S -o /dev/null http://${{ env.APP_ENDPOINT }}/aws-sdk-call/
+          curl -S -o /dev/null http://${{ env.APP_ENDPOINT }}/remote-service?ip=${{ env.REMOTE_SERVICE_POD_IP }}/
+          curl -S -o /dev/null http://${{ env.APP_ENDPOINT }}/client-call/
+
+      - name: Build Gradlew
+        run: |
+          source ${{ env.TEST_RESOURCES_FOLDER }}/.github/workflows/util/execute_and_retry.sh
+          execute_and_retry 2 "./gradlew"
```

Comment:
Instead of this we should just build the validator into a JAR file and run it using that. We can host said JAR file in an S3 bucket on the test accounts and push it via another workflow. This way we do not need to worry about build issues.

Reply:
We can do this in the meantime though if you want to track that as a separate improvement.

Reply:
Will do thanks